pulp_container/app/utils.py

```diff
@@ -225,13 +225,16 @@ def calculate_digest(manifest):
     Calculate the requested digest of the ImageManifest, given in JSON.
 
     Args:
-        manifest (str):  The raw JSON representation of the Manifest.
```

Comment:
This description line was wrong in the original code. Before this commit, the function could only work with bytes.